supply_water.md

```diff
@@ -0,0 +1,112 @@
+
+
+### Water distribution
+
+Conveyance losses and improvements to water distribution efficiencies are included in the water distribution sectors. Conveyance losses for irrigated water use has been included and differentiated for each GCAM region. Conveyance losses/efficiencies for GCAM regions are derived from country level data from [Rohwer et al. 2007](#rohwer2007) and are the weighted mean of the original country level data weighted by irrigated harvested area. 
+
+### Water markets
+
+Water supplies and demands at each basin are balanced through a market mechanism in which prices for water (shadow price) are adjusted until water demands are constrained to available supply. Additional documentation of this capability is available in [Kim et al. 2016](#kim2016), [Turner et al. 2019ab](#turner2019a), [Graham et al. 2018](#graham2018), and [Cui et al. 2018](#cui2018). For additional information, see the detailed description on [water markets](details_#watermarkets).
+
+### Water subsidies and water rights
+
+The water distribution sector for each demand category includes the cost of water transport, distribution and/or treatment and other adjustments such as subsidies, in addition to the resource cost of water. Thus, water prices for each demand category is differentiated. This framework provides a covenient location for including water subsidies or reduced water costs from allocation rights for irrigated agriculture. In GCAM, the price of water for agriculture use has been reduced by a factor of 1/100th. This factor is based on an OECD report on water pricing ([OECD 2009](#oecd2009)) which indicates that industrial and household water users pay more than 100 times as much as agriculture users. In some regions, the marginal cost of irrigated water use is zero. Crop growth requires substantial water inputs and the profitability of agricultural relies on the low cost of water provisions. Without this subsidy, historical calibration of agriculture production in GCAM is not possible for historically water scarce regions with high cost of water. Water subsidies for irrigated agriculture have not been differentiated by region and basin, and improved understanding of regional differences in water subsidies and their impact on agriculture production remain for future work.
+
+## Equations 
+The equations that determine water supply are described here.
+
+#### Variable #1
+
+{Insert equations used to calculate variable #1}
+
+See `method name` in [code_file.cpp](link to code on GitHub).
+
+## Policy options 
```

Comment:
I would recommend removing the policy option section here. We can return to this section once the E4W work is in core and there is some thought on restrictions to pumping groundwater, or a similar constraint placed on the overall extraction of water resources